lib/Language/fr/translation.json

```diff
@@ -297,15 +294,9 @@
       "missingSenapsBaseUrl": "Url de base Senaps non fournie. Vous êtes-vous connecté ?",
       "noStreamsMessageFeatureInfo": "Pas de flux à cet endroit."
     },
-    "mapboxStyle": {
-      "name": "Style Mapbox"
-    },
-    "mapboxVectorTile": {
-      "name": "Mosaïque vectorielle Mapbox"
-    },
-    "mapboxMap": {
-      "name": "Carte Mapbox"
-    },
+    "mapboxStyle": { "name": "Style Mapbox" },
+    "mapboxVectorTile": { "name": "Mosaïque vectorielle Mapbox" },
+    "mapboxMap": { "name": "Carte Mapbox" },
```

Comment:
Can you reformat the translation files to match our other translation files? I think weblate is fine with stringified JSON with 2 spaces.

Reply:
Prettier automatically did that, I have now reverted it to the original 2 spaces format.